### PR TITLE
Feature/course detail page

### PR DESCRIPTION
**Pull request by quebonafide**
typing

### CODE REVIEW

File: client/pages/courses/validate.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export const isPhoneNumber = (value: string, id: string, setErrors: any, errors:
 }
 
 export const isEmail = (value: string, id: string, setErrors: any, errors: string[]) => {
-  const regex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+  const regex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```
Comment (konopek94) [inline client/pages/courses/validate.tsx:29]: What's the purpose of this change? I'm asking, cause I don't see any commit about this change.

**Review by jakpolak — APPROVED** [review]

File: client/pages/courses/validate.tsx (hunk 1 above)
Comment (quebonafide) [reply client/pages/courses/validate.tsx:29]: Lint was telling me to get rid of unnecessary escape characters.

**Review by konopek94 — APPROVED** [review]
